core-feature-pack/galleon-feature-pack/src/main/resources/configs/host/host-primary.xml/config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+
+<config xmlns="urn:jboss:galleon:config:1.0" name="host-primary.xml" model="host">
+    <feature-group name="host-master"/>
```

Comment:
Do you intend in the future to rename this feature-group (plus the host-slave, core-host-master and core-host-slave FGs)? Would it be better to do it now, to remove any chance of having to do another coordinated change to core and full in the future?

Reply:
Hi @bstansberry we have been leaving out the current batch changes that don't affect the users, the reason was to ensure we were able to reach the schedule. However, we are on track with the schedule, there there is a chance we can include them in the current batch.

In any case, we are going to have another "dance" like this one to integrate the test suite, so the current release won't be the last one requiring this kind of integration.

In addition, just for your info, I would need to rebase this on top of https://github.com/wildfly/wildfly-core/pull/5016 to avoid conflicts.

Reply:
@bstansberry I have added the feature group changes, Jobs are progressing well. Let me know if you are fine with them, and I'll rename the latest commit.